definitions/region/countries/countries.yaml

```
@@ -0,0 +1,37 @@
+- Countries:
+    - Austria
+    - Belgium
+    - Bosnia and Herzegovina
+    - Bulgaria
+    - Croatia
+    - Cyprus
+    - Czechia
+    - Denmark
+    - Estonia
+    - Finland
+    - France
+    - Germany
+    - Georgia
+    - Greece
+    - Hungary
+    - Iceland
+    - Ireland
+    - Italy
+    - Latvia
+    - Lithuania
+    - Luxembourg
+    - Malta
+    - Moldova
+    - Montenegro
+    - Netherlands
+    - Macedonia
```

This is to be consistent with the https://github.com/iiasa/pySquirrel package.
```suggestion
    - North Macedonia
```